### PR TITLE
add common_error_reason field to /operations/ response

### PR DESCRIPTION
**Pull request by Tapot**
add field for errors summary

### CODE REVIEW

File: operations.openapi.json [hunk 1]
```diff
@@ -128,6 +128,10 @@
           "error": {
             "$ref": "#/components/schemas/JobError"
           },
+          "common_error_reason": {
+            "example": "One or more of async operations return an error",
```
Comment (tyanas) [inline operations.openapi.json]: How it differs from the `error` field above? @Tapot

Comment (Tapot) [reply operations.openapi.json]: error - list of providers errors, if we have an error before any provider request, we have null value for `error` field
common_error_reason - common error message, a string as summary or all errors
See docs - https://intento.atlassian.net/wiki/spaces/IN/pages/250445980/Operations

Comment (tyanas) [reply operations.openapi.json]: а ты можешь описать что error это список? @Tapot 
сейчас оно описано как объект
https://github.com/intento/intento-api/blob/feature/DEV-55_add_new_field_to_operations_openapi_description/operations.openapi.json#L201-L203

Comment (Tapot) [reply operations.openapi.json]: it's my bad, error is an object, errors are grouped by provider id
but I fixed 'type' field name - 'intent' is correct name

**Pull request merged**